README.md

```diff
@@ -252,8 +252,8 @@ contribute:
   filing a new issue to discuss the proposed change, and then submit a
   pull request to address the issue. `rnassqs` follows the style
   outlined in Hadley Wickham’s [R
-  Packages](https://r-pkgs.org/r.html#code-style). Following this style
-  makes the pull request and review go more smoothly.
+  Packages](https://r-pkgs.org/code.html#code-style). Following this
```

Comment:
I see you set a new line after a given number of characters.
A few years ago I started adding new lines after periods or parts of sentences instead, because it makes diffs easier to read: https://cirosantilli.com/markdown-style-guide/#line-wrapping

I'm mentioning this because I've found the change very helpful (once I took the habit :zany_face:)